csm/conf/setup.py

```diff
@@ -796,7 +815,10 @@ def config(self, args):
             if not self._replacement_node_flag:
                 self.Config.create(args)
             self.Config.load()
-            UDSConfigGenerator.apply(uds_public_ip=uds_public_ip)
+            is_vm = Setup._is_vm()
```

Comment:
Lyve pilot should be configured regardless of VM or not. The only check we should have is to confirm if lyve_pilot is supported or not.

         `If Lyve pilot is supported
                configure lyve pilot
          else
                 don;t configure lyve pilot`

Reply:
OK then.  I will remove it.

Reply:
Done.  Tested on OVA setup provided by Gaurav with the corresponding changes to `unsupported_features.json` being done manually.